### PR TITLE
Insert group name into services of so11y and istio relevant metrics

### LINKED ISSUE #5853: Add group to the self-observability dashboard configuration
Simply follow this #5851

### PR DESCRIPTION
**Pull request by hanahmily**
### Introduce service group into MAL related feats
- [x] Update the documentation to include this new feature.
- [x] Tests(including UT, IT, E2E) are added to verify the new feature.
- [x] If it's UI related, attach the screenshots below.
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #5853 .
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ Release Notes.
 * Fix "transaction too large error" when use TiDB as storage.
 * Add otel rules to ui template to observe Istio control plane.
 * Remove istio mixer
+* Insert group name into services of so11y and istio relevant metrics
 
 #### UI
 * Fix incorrect label in radial chart in topology.
```
Comment (wu-sheng) [inline CHANGES.md:60]: Please update the changelog for UI to include all updates, NOTE, there is a new update in UI(bug fix), please re-sync.

Comment (hanahmily) [reply CHANGES.md:60]: done

File: CHANGES.md [hunk 2]
```diff
@@ -54,10 +54,12 @@ Release Notes.
...
 * Replace node-sass with dart-sass.
+* Replace serviceFilter with serviceGroup
```
Comment (wu-sheng) [inline CHANGES.md:62]: UI update includes
1. https://github.com/apache/skywalking-rocketbot-ui/pull/383
1. https://github.com/apache/skywalking-rocketbot-ui/pull/367

Comment (hanahmily) [reply CHANGES.md:62]: done

**Review by wu-sheng — APPROVED** [review]
LGTM. Do I need to redeploy the demo ENV?

**Comment by hanahmily** [comment]
> LGTM. Do I need to redeploy the demo ENV?

IIRC, demo actives so11y. if that, yes, we should redeploy it. the new service name is like below:
WX20201126-061521@2x

**Comment by wu-sheng** [comment]
> IIRC, demo actives so11y. if that, yes, we should redeploy it. the new service name is like below:

We need to move it to the GCP, as the donated env is going to be gone soon.

**Comment by wu-sheng** [comment]
Due to self-observability changed, you need to fix the e2e as OAP service name changed with that.

File: test/e2e/e2e-test/src/test/java/org/apache/skywalking/e2e/PythonE2E.java [hunk 3]
```diff
@@ -101,7 +101,7 @@ public void tearDown() {
     @RetryableTest
     void services() throws Exception {
         List<Service> services = graphql.services(new ServicesQuery().start(startTime).end(now()));
-        services = services.stream().filter(s -> !s.getLabel().equals("oap-server")).collect(Collectors.toList());
+        services = services.stream().filter(s -> !s.getLabel().equals("oap::oap-server")).collect(Collectors.toList());
```
Comment (wu-sheng) [inline test/e2e/e2e-test/src/test/java/org/apache/skywalking/e2e/PythonE2E.java:104]: @kezhenxu94 As a reminder, please consider to don't hardcode this kind of thing in the next-generation e2e framework. This is too tricky.

Comment (hanahmily) [reply test/e2e/e2e-test/src/test/java/org/apache/skywalking/e2e/PythonE2E.java:104]: My preference is, so11y test cases should leverage the new `group` field to filter services in https://github.com/apache/skywalking-query-protocol/blob/d3608e86f7c9aaf54aa4ad2c8d2afd09d0680dd6/metadata.graphqls#L72.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**